2_process/PrepForViz.R

```diff
@@ -0,0 +1,13 @@
+#Set the working directory to the path where this script is stored.
+setwd(dirname(rstudioapi::getActiveDocumentContext()$path))
+
+source("src/format_functions.R")
+
+inPath <- "C://Projects/ds-pipelines-targets-1-course-static/1_fetch/out"
+infile <- file.path(inPath,"model_RMSEs.csv")
+out.prepped <- read.format.infile(infile)
+outPath <- "C://Projects/ds-pipelines-targets-1-course-static/2_process/out"
+outfile <- file.path(outPath,"RMSE_prepped.csv")
+
+#Write the prepped file
+out.csv(out.prepped,outfile)
```

Comment:
I don't think this function is adding much value. It's a wrapper around `readr::write_csv` that uses the exact same arguements, so I think it just adding processing time.

```
> toc()                                                                       
0.01 sec elapsed
> tic()
> out.csv(out.prepped,outfile2)
> toc()                                                                       
0.02 sec elapsed
> tic()
> out.csv(out.prepped,outfile2)
> toc()                                                                       
0 sec elapsed
> tic()
> out.csv(out.prepped,outfile2)
> toc()                                                                       
0.02 sec elapsed
> readr::write_csv(out.prepped,outfile2)
                                                                              
> readr::write_csv(out.prepped,outfile2)
                                                                              
> readr::write_csv(out.prepped,outfile2)
```

Reply:
You're right. I think I was following a pattern because of setting everything to be explicitly with functions for the exercise before moving to targets implementation. Corrected